test/support/utils.ts

```diff
@@ -59,3 +59,32 @@ export function promisifyChildProcess(
     })
   })
 }
+
+export function waitUntil(
+  expectation: () => void,
+  { timeout = 3000, interval = 1000 } = {},
```

Comment:
We are not using these options at the moment, so we can drop them from being in the arguments.

```ts
const TIMEOUT = 3000
const RETRY_INTERAL = 1000

export function waitUntil(expectation) { ... }
```